YAXLibTests/SerializationTest.cs

```diff
@@ -113,26 +117,44 @@ public void BookWithDecimalPriceTest()
         public void CultureChangeTest()
         {
             var curCulture = CultureInfo.CurrentCulture;
-
-            Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo("fr-FR");
-            var serializer = new YAXSerializer(typeof(CultureSample), YAXExceptionHandlingPolicies.DoNotThrow, YAXExceptionTypes.Warning, YAXSerializationOptions.SerializeNullObjects);
+#if FXCORE
+			CultureInfo.CurrentCulture = new CultureInfo("fr-FR");
+#else
+			Thread.CurrentThread.CurrentCulture = new CultureInfo("fr-FR");
+#endif
+			var serializer = new YAXSerializer(typeof(CultureSample), YAXExceptionHandlingPolicies.DoNotThrow, YAXExceptionTypes.Warning, YAXSerializationOptions.SerializeNullObjects);
             string frResult = serializer.Serialize(CultureSample.GetSampleInstance());
 
-            Thread.CurrentThread.CurrentCulture = CultureInfo.GetCultureInfo("fa-IR");
-            serializer = new YAXSerializer(typeof(CultureSample), YAXExceptionHandlingPolicies.DoNotThrow, YAXExceptionTypes.Warning, YAXSerializationOptions.SerializeNullObjects);
+#if FXCORE
+			CultureInfo.CurrentCulture = new CultureInfo("fa-FR");

```

Comment:
The typo does not influence the test result because date formats are the same, but it was a bug.